README.md

```diff
@@ -358,12 +358,13 @@ There are open issues in the microsoft vscode repository that block the ability
     "test",
     "--runInBand",
     "--no-cache",
-    "--env=jsdom-fourteen",
+    "--env=jsdom",
     "--transformIgnorePatterns",
     // https://github.com/microsoft/vscode/issues/81944
```

Comment:
Not sure why this is highlighted as red.  That makes me think this part of the string was deleted, which it wasn't.

Reply:
The red highlight is because it's invalid JSON syntax (as far as GitHub's markdown parser can tell). `launch.json` allows JSON-with-comments so it's not a real issue.